e2e/templates/agent/deployment.yaml

```diff
@@ -18,15 +18,18 @@ spec:
       containers:
       - name: konnectivity-agent-container
         image: {{ .Image }}
+        securityContext:
+          allowPrivilegeEscalation: false
+          runAsUser: 0
```

Comment:
Why are we running as the root user?

Reply:
I was copying the pre-existing example in `kind/`. I believe it is necessary for access to the kubeconf in `/etc/kubernetes`.